central/src/components/CreateQuestion.jsx

```diff
@@ -94,9 +94,30 @@ export default function QuestionForm({ updateQuestion, question: initialState, g
   // Handles grade, domain, cluster, or standard change/update
   const onSelectMaker = useCallback((field) => ({ target }) => { setQuestion({ ...question, [field]: target.value }); }, [question, setQuestion]);
 
+  const handleAddQTToGT = async (question) => {
+    console.log(addQToGT("a", "b"));
```

Comment:
hmm, might worth to start adopting different layers of logging, `info` vs. `debug` vs. `error` vs. `warn` [reference](https://stackoverflow.com/a/36355027/1450348)

Reply:
good idea, I'll review and implement moving forward. This specific function has since been removed.